src/tests/ftest/nvme/nvme_pool_extend.py

```diff
@@ -136,82 +134,85 @@ def run_ior_thread(self, action, oclass, api, test):
                                            "test": test,
                                            "flags": flags,
                                            "results":
-                                           self.out_queue})
+                                               self.out_queue})
         # Launch the IOR thread
         process.start()
         # Wait for the thread to finish
         process.join()
 
-    def run_nvme_pool_extend(self, num_pool):
+    def create_pool(self):
+        """ Creates a TestPool based on yaml params.
+
+        Returns:
+            TestPool: Created pool
+        """
+        # Create a pool
+        pool = TestPool(self.context, dmg_command=self.dmg_command)
+        pool.get_params(self)
+        # Split total SCM and NVME size for creating multiple pools.
+        pool.scm_size.value = int(pool.scm_size.value)
+        pool.nvme_size.value = int(pool.nvme_size.value)
```

Comment:
What's the purpose of converting the size to int? Isn't it already int? Also don't understand the comment on line 152.

Reply:
Right, the comment is outdated. Since the code was changed to use a pool per iteration, it is not needed to split the size, the split was done by size / num_pools.
Regarding the casting to int, I could try without it.

Reply:
It is already an int, no need to cast it.